docs/dm/assets.md

```diff
@@ -87,62 +98,72 @@ This menu includes a plethora of options.
 
 ### Is invisible
 
-Selecting this option turns the asset invisible to your players.
-As the DM, you will still be able to see the asset.
+Selecting this option turns the shape invisible to your players.
+As the DM, you will still be able to see the shape.
 This can be useful for hiding future encounters or placing enemies in stealth. 
 
+Players with vision or edit access will still see the object with this enabled.
+There is currently no special indicator that shows it's invisible to other users. 
+
 ### Is locked
 
-Selecting this option locks the object in place.
+Selecting this option locks the shape in place.
 This can be useful for maps you don’t want to accidently move while playing.
+Press ```Cntrl + L``` to do this without opening the Edit asset menu.
 
 ### Show badge
 
-Selecting this option displays a badge at the bottom right of the object.
+Selecting this option displays a badge at the bottom right of the shape.
 
 ### Blocks vision/light
 
-Selecting this option blocks vision and light from passing through the object.
+Selecting this option blocks vision and light from passing through the shape.
 More information about lighting can be found in the [lighting and vision](/docs/dm/light-shadows/) document.
 
 ### Blocks movement
 
-Selecting this option prevents other objects from moving through it.
+Selecting this option prevents other shapes from moving through it.
 This is useful for creating walls in your dungeon you don’t want your players to pass through.
 
 ### Border colour & Fill colour
 
-These options control the color of the borders around and the fill color of default tokens. 
+These option applies to basic tokens.
+Border colour controls the colour of the borders around the basic token
+Fill colour of controls the colour of the inside of the basic token.
 
 ### Access
 
-Here is listed the players who have access to the asset.
-Selecting a player name and “add access” will allow them to interact with the asset.
-Selecting any of the icons next to “default” gives all players access to the asset.
-The pen icon allows those with access to edit the asset.
-The four arrows icon allows those with access to move the asset unless locked.
-The lightbulb icon allows those with access to see the asset. 
+Here is listed the players who have access to the shape.
+Selecting a player name and “add access” will allow them to interact with the shape.
+Selecting any of the icons next to “default” gives all players access to the shape.
+The pen icon allows those with access to edit the shape.
+The four arrows icon allows those with access to move the shape unless locked.
+The lightbulb icon allows those with access to see the shape. 
 
 ### Trackers
 
-Trackers can track numerical values of assets, such as health to player tokens.
+Trackers can track numerical values of shapes, such as health to player tokens.
```

Comment:
```suggestion
Trackers can track various values of shapes, such as health to player tokens, or attitude to NPC.
```
Any input seems to be accepted. This is, in my opinion, very much wanted behaviour, as you can e.g. track NPC's attitude (hostile/friendly etc.) or whatever else, without needing a table to cross-reference numerics. :)